main/src/ui/chat_input/chat_input_controller.vala

```diff
@@ -68,7 +88,20 @@ public class ChatInputController : Object {
         input_field_status = status;
 
         chat_input.set_input_state(status.message_type);
-        status_description_label.label = status.message;
+        if (status.contains_markup){
+            status_description_label.set_markup("%s".printf(Markup.escape_text(status.message)));
```

Comment:
As you overwrite the label in the very next line, this is equivalent to just setting `use_markup` to `true`. Please do that instead:
```vala
status_description_label.use_markup = true;
```